src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSortVariant.unit.spec.jsx

```diff
@@ -175,11 +173,13 @@ describe('VAOS ProviderSortVariant on <CommunityCareProviderSelectionPage>', ()
         selector: 'button',
       }),
     );
-    fireEvent.change(await screen.getByLabelText('Show providers closest to'), {
-      target: {
-        value: FACILITY_SORT_METHODS.distanceFromCurrentLocation,
-      },
+
+    const sel = await screen.findByTestId('providersSelect');
```

Comment:
This is minor. Make the constant name with meaningful context. Maybe like 'select' or 'dropdown'